packages/portfolio-contract/src/pos-gmp.flows.ts

```diff
@@ -149,7 +157,7 @@ harden(CCTP);
 
 export const sendGMPContractCall = async (
   dest: { axelarId: string; remoteAddress: EVMT['address'] },
-  calls: ContractCall[],
+  payload: number[],
```

Comment:
that looks like a significant regression. I hope to study the motivation further

Reply:
does `buildNoncePayload` not build a list of calls?

Reply:
what is `buildNoncePayload` building that is not a list of calls? Doesn't the nonce payload represent calls to our factory contract?

I guess I should read more closely...